recipes/windows_service.rb

```diff
@@ -20,6 +20,7 @@
 
 return unless platform? 'windows'
 
-service 'WinRM' do
+windows_service 'WinRM' do
```

Comment:
Now you have to update the chefspec test, sorry ><

Reply:
Fix notify block.